packages/wonder-blocks-modal/components/two-column-modal.js

```diff
@@ -45,6 +57,12 @@ const styles = StyleSheet.create({
         overflow: "hidden",
     },
 
+    closeButton: {
+        position: "absolute",
+        left: 24,
+        top: 16,
```

Comment:
Once we use the 48x48 bounding box for the close button, this should be `left: 4, top 8`.

Reply:
Done!